### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by thurs8**
* Add the correct save_name variable to the **convert_to_jit** func (the script was able to fall without this variable)
* fix metric name for callbacks monitor parameter 
* add correct way to use several torchmetrics (I had a problem with calculating metrics on several GPUs. For more details see the discussion: https://github.com/PyTorchLightning/pytorch-lightning/issues/4909#issuecomment-736821581)

### CODE REVIEW

File: src/lightning_classes/lightning_image_classification.py [hunk 1]
```diff
@@ -13,22 +13,14 @@ def __init__(self, cfg: DictConfig):
...
+        })
         if 'other_metrics' in self.cfg.metric.keys():
             for metric in self.cfg.metric.other_metrics:
-                self.metrics.append(
-                    {
-                        'metric': load_obj(metric.class_name)(**metric.params).to(self.cfg.general.device),
-                        'metric_name': metric.metric_name,
-                    }
-                )
+                self.metrics.update({
```
Comment (Erlemar) [inline src/lightning_classes/lightning_image_classification.py:21]: If `self.metrics` is changed, than it is necessary to update the code for logging metrics in train and validation steps.
